### ISSUE TITLE
Mucous diarrhea

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6404]
**Preferred term label:**
Mucous diarrhea

**Synonyms**
Mucoid diarrhea

**Definition (free text, please give PubMed ID)**
Passage of many stools containing mucus.
PMID: 31869107

**Parent term (use hpo.jax.org/app)**
Diarrhea

**Comment by pnrobinson** [comment]
added term
